### ISSUE TITLE
Citation Dialog: Reference Box does not disappear when the dialog loses focus.

### ISSUE THREAD

**Issue by adomasven** [zotero/zotero#3627]
I've seen this before, and now with the reference box opening automatically it's a real problem.

Report https://forums.zotero.org/discussion/111279/7-0-0-beta-56-9edfcba9a-citation-picker-opens-behind-libreoffice-window-only-dropdown-visible

**Comment by adomasven** [comment]
> I wonder if it would make sense to look into making a reference panel not a <panel> but just a <vbox> (or something similar) just sitting inside of the main window below the editor? That should allow us to not worry about the positioning at all: from hardcoded margins to noautofocus/noautohide to this issue of the reference panel being separated from the main dialog.

I think it's a <panel>, because a regular <vbox> cannot be positioned outside of the window, but regardless, there's probably a way to make it not appear on top of other windows on Linux.

**Comment by abaevbog** [comment]
> I think it's a , because a regular cannot be positioned outside of the window, but regardless, there's probably a way to make it not appear on top of other windows on Linux.

Right. What I was trying to say is that we already have to handle window resizing back and forth due to the editor's height changing. So if there was a benefit to getting rid of the `panel`, we could place the reference panel fully inside of the window and have the resizing handle not just the editor but also the variability of the reference panel. This is a (obviously not at all working) screenshot of what I mean conceptually.

<img width="1483" alt="Screenshot 2024-03-01 at 11 26 12 AM" src="https://github.com/zotero/zotero/assets/36271954/27a95e95-69d6-49cd-a853-c0a13d902fa0">

This may not be the actual solution to the windows issue though and may or may not be worth it

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/112591/i-can-not-insert-a-citation-in-word
